authorization/src/main/java/com/terabite/authorization/repository/LoginRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.terabite.authorization.repository;
+
+import com.terabite.authorization.model.Login;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+
+public interface LoginRepository extends JpaRepository<Login, Long> {
+    Login findByEmail(String email);
+
+    Login findByPasswordResetToken(String passwordResetToken);
```

Comment:
In my testing it seems like instead of crashing the program these simply return void if the target is not found. That being said, I still have error catching everywhere these are used just in case.